.github/workflows/release_serialization.yml

```diff
@@ -0,0 +1,104 @@
+name: Serialization Library Release
+
+on: workflow_dispatch
```

Comment:
Is this saying it's now a manual trigger?

Reply:
Yes